src/Google.Api.Gax.Grpc/PageStreamingAsync.cs

```diff
@@ -43,7 +44,12 @@ public sealed class PagedAsyncEnumerable<TRequest, TResponse, TResource> : IPage
         }
 
         /// <inheritdoc/>
-        public IResponseAsyncEnumerable<TResponse, TResource> AsPages() => _pages;
+        public IAsyncEnumerable<TResponse> AsResponses() => _pages;
+
+        /// <inheritdoc/>
+        public Task<FixedSizePage<TResource>> GetCompletePageAsync(
```

Comment:
It's complete in terms of "it'll give you everything you asked for if it possibly can". But I definitely want a better name :)